### PR TITLE
Unit Test Fix for Report Service

### PR DESCRIPTION
**Pull request by jayfyle**

### CODE REVIEW

File: src/app/core/services/report.service.spec.ts [hunk 1]
```diff
@@ -747,6 +744,46 @@ describe('ReportService', () => {
...
+        },
+      };
+
+      const getMyReportsParam1 = {
+        offset: 0,
+        limit: 2,
+        queryParams: {
+          rp_state: 'in.(DRAFT,APPROVER_PENDING,APPROVER_INQUIRY)',
+        },
+        order: undefined,
```
Comment (rvab) [inline src/app/core/services/report.service.spec.ts:762]: if it's `undefined`, then do you still need to pass it?

File: src/app/core/services/report.service.spec.ts [hunk 2]
```diff
@@ -747,6 +744,46 @@ describe('ReportService', () => {
...
+      };
+
+      const getMyReportsParam2 = {
+        offset: 2,
+        limit: 2,
+        queryParams: {
+          rp_state: 'in.(DRAFT,APPROVER_PENDING,APPROVER_INQUIRY)',
+        },
+        order: undefined,
```
Comment (rvab) [inline src/app/core/services/report.service.spec.ts:771]: same as above

**Review by rvab — APPROVED** [review]
One minor doubt

File: src/app/core/services/report.service.spec.ts (hunk 1 above)
Comment (jayfyle) [reply src/app/core/services/report.service.spec.ts:762]: Yes, otherwise the test fails, it is an expected property in the object

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**